check.py

```diff
@@ -243,7 +243,7 @@ def minify_check(wast, verify_final_result=True):
 
 not_executable_suffix = ['.txt', '.js']
 executables = sorted(filter(lambda x: not any(x.endswith(s) for s in
-                                              not_executable_suffix),
+                                              not_executable_suffix) and os.path.isfile(x),
```

Comment:
Yes, the `FILE(COPY)` command below adds new directories under `bin/`, for example `bin/js`.